### ISSUE TITLE
Include the deprecation message in NoDeprecated's details

### ISSUE THREAD

**Issue by jfmengels** [jfmengels/elm-review-common#17]
Currently, when reporting an error the `NoDeprecated` rule includes the following as its details:

```
This element was marked as deprecated and should not be used anymore.

Please check its documentation to know the alternative solutions.
```

One way that things are tagged as deprecated is when the module or function/type contains `@deprecated` in their documentation. What we could try to do is to extract the deprecation message next to that marker, and include it in the details when available.

**Comment by absynce** [comment]
Yes! I've wanted this feature but I was too lazy to ask. 😊 Thanks for bringing it up! 🙌 

Right now, if anyone on the team encounters this on a module/function/type, they usually come to me to figure out what to do instead of reading the docs. If the error included the message from the docs, it might reduce the back and forth. It would reinforce even more that Elm error messages are meaningful and worth the time to read. 

Here's an example of what I currently have for a deprecated module & function:
```
module Util.Decoders exposing (decodeErrors_DEPRECATED)

{-| This is deprecated.

@deprecated Use `Api.viewError` instead.

Move any reusuable decoders to their types. Examples:

  - `Address.decoder`
  - `Money.decoder`
  - `TimeOfDay.decoder`

-}

import Dict
import Json.Decode as JD
import List.Extra as Extra


{-| @deprecated Use `Api.viewError` instead, it will do this for you.
-}
decodeErrors_DEPRECATED : (String -> String) -> String -> List ( String, String )
decodeErrors_DEPRECATED toKey val =
    ...
```
